### PR TITLE
OP_LOADSYM

### PR DESCRIPTION
**Pull request by xor256**
Symbol must be read from global symbol table or from irep ?
I m don't know this will be working.

### CODE REVIEW

File: doc/bytecode_format.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
 | 06 | nregs | 2 | binary | number of register variable |
 | 08 | rlen | 2 | binary | number of child irep ? |
 | 0A | ilen | 4 | binary | number of instruction |
-| 0E | irep | (ilen)*4 | bunary |  instructions |
```
Comment (vessi) [inline doc/bytecode_format.md]: can you please explain what is `bunary`?

File: doc/bytecode_format.md [hunk 2]
```diff
@@ -40,7 +40,7 @@
...
+| 0E | irep | (ilen)*4 | binary |  instructions |
```
Comment (vessi) [inline doc/bytecode_format.md]: sorry previous comment was wrong

Comment (xor256) [inline doc/bytecode_format.md]: the problem is when ilen=0 
what happened?

**Comment by kaz0505** [comment]
Thank you.

ilen == 0 is not assumed. 
ilen shows the number of instructions in a method.
Even if an empty method, OP_RETURN is exist.

**Comment by xor256** [comment]
thanks

**Pull request merged**